selfdrive/car/tests/test_can_fingerprint.py

```diff
@@ -11,13 +11,11 @@ def test_can_fingerprint(self, car_model, fingerprints):
     """Tests online fingerprinting function on offline fingerprints"""
 
     for fingerprint in fingerprints:  # can have multiple fingerprints for each platform
-      can = messaging.new_message('can', 1)
-      can.can = [log.CanData(address=address, dat=b'\x00' * length, src=src)
-                 for address, length in fingerprint.items() for src in (0, 1)]
+      can = [CanData(address=address, dat=b'\x00' * length, src=src)
```

Comment:
I discovered a bug in the test! it was feeding it an empty can list which get_one_can never usually would return!